pages/series/index.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ *  @file [series].tsx
+ * Landing page for a series, list of recent episodes
+ *
+ */
+import styles from '@styles/Series.module.scss';
+import PrxLogo from '@svg/prx-logo.svg';
+
+const SeriesPage = () => {
+  console.log();
+  return (
+    <div>
+      <div className={styles.root}>
+        <div className={styles.logoBar}>
+          <PrxLogo className={styles.prxLogo} />
+        </div>
+        <div className={styles.header}>{/* Header element */}</div>
+
+        <div className={styles.episodeContainer}>
+          <h2 className={styles.content}>Latest Episodes</h2>
```

Comment:
Create a new style named `heading`. Only set `margin-block: 2rem` to add the spacing around it.